genesis/public/core/class_test.go

```diff
@@ -29,5 +32,181 @@ func TestClassDomain_GetClassID(t *testing.T) {
 	assert.NoError(t, err)
 
 	domainID := clsDom.GetClassID()
+
 	assert.Equal(t, class.ClsDomainID, domainID)
 }
+
+func TestClassDomain_GetClass_NoSuchRecord(t *testing.T) {
+	parent := &mockParent{}
+	classDom, err := newClassDomain(parent)
+	assert.NoError(t, err)
+	cl, err := classDom.GetClass("test")
+
+	assert.Error(t, err)
```

Comment:
It's better to use `assert.EqualError(t, err, <some_error_message>)` and check error message too.